eav/migrations/0004_auto_20170124_1011.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you merge your migration files (0004_auto_20170124_1011, 0003_auto_20161104_0922) into a single one?